### ISSUE TITLE
Documentation. Setting auth type

### ISSUE THREAD

**Issue by plutocrat** [untitaker/vdirsyncer#282]
In your documentation you say
"Set the auth parameter of vdirsyncer.storage.CaldavStorage, vdirsyncer.storage.CarddavStorage, and/or vdirsyncer.storage.HttpStorage to basic or digest (not guess)."
I've been trying to figure out how to do this for the last three hours. Is this a modification in the code? Command line? Config file? Just a small hint would be good. 
Also pip install method didn't work ... still got the wrong version of the python-requests. 
Also pipsi install method just wouldn't install at all. Not your problem, I know. 
So basically I can't get this to work on Ubuntu at all by any of the means suggested in the documentation.

**Comment by untitaker** [comment]
The auth parameter in your storage sections. If you click on either of those links, you'll see a config example for each storage.

On 16 October 2015 11:32:02 CEST, plutocrat <notifications@github.com> wrote:
>In your documentation you say
>"Set the auth parameter of vdirsyncer.storage.CaldavStorage,
>vdirsyncer.storage.CarddavStorage, and/or
>vdirsyncer.storage.HttpStorage to basic or digest (not guess)."
>I've been trying to figure out how to do this for the last three hours.
>Is this a modification in the code? Command line? Config file? Just a
>small hint would be good. 
>
>---
>Reply to this email directly or view it on GitHub:
>https://github.com/untitaker/vdirsyncer/issues/282

-- 
Sent from my phone. Please excuse my brevity.

**Comment by plutocrat** [comment]
Clicked on a thousand links before. Just clicked on them all again, and still can't find any information on this. So basically you just mean I need a line saying 
auth = basic 
in the [storage blah-blah] section of the ~/.vdirsyncer/config file.

**Comment by untitaker** [comment]
Ah sorry, yes, the links in the changelog are broken atm, see https://github.com/untitaker/vdirsyncer/issues/243.

If you take a look at https://vdirsyncer.readthedocs.org/en/stable/config.html#storage-section, you'll see those identifiers again.

**Comment by plutocrat** [comment]
Wow, OK, that was easy in the end. I was searching through the source code to find the answer! Ran into another few problems after that with SSL but they don't belong here. Eventually synced everything and all is good, concluding a week long struggle to copy data from one owncloud installation to another. Thanks for this tool.
